mailpoet/package.json

```diff
@@ -29,7 +29,7 @@
   "dependencies": {
     "@babel/runtime": "^7.22.6",
     "@babel/runtime-corejs3": "^7.22.6",
-    "@emotion/react": "^11.8.2",
+    "@emotion/react": "^11.11.1",
```

Comment:
Note to self to check later — why do we even install `@emotion` packages explicitly? 🤔